### ISSUE TITLE
Crashes as soon as I press a key

### ISSUE THREAD

**Issue by hkirat** [randomboy24/mouse_tracker_2#2]
![Image](https://github.com/user-attachments/assets/b5b93cbd-8394-4b4f-96e8-02991fc48926)

**Comment by randomboy24** [comment]
The app works fine on my Linux machine. On Mac, it's likely crashing due to missing permissions. Try enabling Accessibility & Input Monitoring for the app in System Settings , Privacy & Security. That should fix it.

**Comment by randomboy24** [comment]
It doesn't crash on key press on my machine, although it won't record key presses because I forgot to add that feature. (I've added it in the recent commit.) Other than that, it should be fine. It's probably crashing due to a compatibility issue with macOS. Try running it on Linux , it works there.
